pkg/api/extensions/accessor.go

```diff
@@ -147,6 +147,18 @@ func (u unstructuredStatusAccessor) GetLastOperation() *gardencorev1beta1.LastOp
 	return lastOperation
 }
 
+// SetLastOperation implements Status.
+func (u unstructuredStatusAccessor) SetLastOperation(lastOp *gardencorev1beta1.LastOperation) {
```

Comment:
I don't have a good answer because this code was contributed by somebody who left the team, but I guess it's not too critical because I think this entire code might not be even used at all. From https://github.com/gardener/gardener/blob/77e09dd522d7bfb682cc35d0be437201b606061b/pkg/api/extensions/accessor.go#L33-L42, I understand that we always work with the dedicated Golang type instead of `unstructured` because all our extension objects implement the `extensionsv1alpha1.Object`.
Most likely, we can remove this unstructured accessor in a subsequent PR.

Reply:
thanks, makes sense.